examples/airline/configs/agents.py

```diff
@@ -0,0 +1,93 @@
+from configs.tools import *
+from data.routines.baggage.policies import *
+from data.routines.flight_modification.policies import *
+from data.routines.prompts import STARTER_PROMPT
+
+from openai_runner import Agent
+
+
+def transfer_to_flight_modification():
+    """Call this function when a user needs to be transferred to flight modification agent."""
+    return flight_modification
```

Comment:
Consider using a more robust error handling strategy in function definitions to manage exceptions and provide feedback to the calling functions. [medium]